src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/UserDefinedClientFeatures.java

```diff
@@ -29,11 +31,44 @@ public UserDefinedClientFeatures() {
         setCompletionFeature(new UserDefinedCompletionFeature());
         setFormattingFeature(new UserDefinedFormattingFeature());
         setWorkspaceSymbolFeature(new UserDefinedWorkspaceSymbolFeature());
+        setEditorBehaviorFeature(new UserDefinedEditorBehaviorFeature(this));
     }
 
     public boolean isCaseSensitive(@NotNull PsiFile file) {
-        UserDefinedLanguageServerDefinition serverDefinition = (UserDefinedLanguageServerDefinition) getServerDefinition();
+        ClientConfigurableLanguageServerDefinition serverDefinition = (ClientConfigurableLanguageServerDefinition) getServerDefinition();
```

Comment:
This was not previously changed to `ClientConfigurableLanguageServerDefinition` but is being configured in tests now, so it needed to be.